packages/block-library/src/image/image.js

```diff
@@ -770,7 +769,9 @@ export default function Image( {
 				} }
 				resizeRatio={ align === 'center' ? 2 : 1 }
 			>
-				{ img }
+				{ /* If the image has a href, wrap in an <a /> tag to trigger any inherited link element styles */ }
+				{ /* eslint-disable-next-line jsx-a11y/anchor-is-valid */ }
```

Comment:
Can we remove this line and add `href="#"` instead? Then you can disable the link click as my PR shows.

https://github.com/WordPress/gutenberg/pull/54322

This method has been implemented for a couple different blocks now. Trying to match styles is not a good reason to have an invalid link markup.

Reply:
Thanks, I was just coming back to this PR to ask whether the single `<a />` was sufficiently accessible. Good to have my question answered in advance! Thank you!